### PR TITLE
Add docker

### PR DESCRIPTION
**Pull request by kmagida110**
## Background
Add dockerfile to containerize data extraction

### CODE REVIEW

File: run_extraction.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+if [ -z $1 ]
+  then
+    echo "Missing arguments, usage ./run_extraction.sh aws_mfa_arn MFA_CODE"
+    exit 1
+fi
```
Comment (bamader) [inline run_extraction.sh:7]: Lol I haven't seen a good ol' `fi` in a while

**Review by bamader — APPROVED** [review]
LGTM!

**Comment by bamader** [comment]
Code looks good overall. One general question, though: why do we keep going back to edit / revise the geocode stuff? Not in the sense of like "oh we shouldn't be spending our time on that" (because we def should), more from the perspective of I'm not someone who's ever worked with geocoding data before. Is it like inherently more tricky than other work we've done? It basically seems like every PR we put up adds geocoding stuff or changes geocoding stuff and I'm not exactly sure what's so complex or different about it that that's the case.

**Comment by kmagida110** [comment]
@bamader The only tangible change for geocoding here was reducing the file size that we load to the census from 10,000 rows to 2,000 and that was 100% because I wanted to see the progression through the files. The 10,000 row files takes as long as five 2,000 row files, which is ~15 minutes and I want to know that it is running during that time!

**Comment by kmagida110** [comment]
@bamader Most of the other geocoding changes were conforming the output to match other changes I was making and not geocoding in particular.

**Comment by bamader** [comment]
Gotcha. Output conforming would explain things. There was a lot of what seemed like pretty small name changes (like with legislative lookups and or lat_long things) and I was like what's going on here, but if it's just to get things formatted right that makes sense.

**Review by imcurtis — APPROVED** [review]

File: set_aws_session.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```
Comment (JordanGuinn) [inline set_aws_session.sh:1]: Wow THANK YOU for doing this - definitely gonna steal it and throw it in the `ctoec-devops` repo as well

**Review by JordanGuinn — APPROVED** [review]
I don't have much an opinion on the python changes here to be completely honest, but the Docker updates seem fine for our purposes 👍👍

**Pull request merged**